tox.ini

```diff
@@ -1,14 +1,23 @@
 [tox]
 
 envlist =
-    flake8
-    mypy
-
+    flake8, black, mypy
     coverage-py{27,35,36,37,38,py2,py3}-tw{171,184,current,trunk}
-
+    coverage_report
     docs, docs-linkcheck
+    packaging
+
+skip_missing_interpreters = {tty:True:False}
+
+
+[default]
```

Comment:
This environment holds values that can be reused later, allowing use to set them in one place.